src/partisan_config.erl

```diff
@@ -29,11 +29,36 @@
          get/2]).
 
 init() ->
-    %% Generate a random peer port.
-    rand_compat:seed(erlang:phash2([node()]),
-                     erlang:monotonic_time(),
-                     erlang:unique_integer()),
-    RandomPeerPort = rand_compat:uniform(1000) + 10000,
+    %% override Erlang env configuration with OS var configurations if set
+    IPAddress = case os:getenv("IP", "false") of
+                    "false" ->
+                        ?PEER_IP;
+                    IP ->
+                        {ok, ParsedIP} = inet_parse:address(IP),
+                        application:set_env(partisan, peer_ip, ParsedIP),
```

Comment:
Do these also need to be set in partisan_config as well?  I think parts of the Lasp suite might read them...